docs/pages/guides/inviting-member.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Inviting team members'
+---
+
+import { Steps, Callout } from 'nextra-theme-docs'
+
+# Team Member Invitation and Role Management in Liteflow
+
+Building a team within the Liteflow platform allows for effective management and delegation of tasks, ensuring that your platform operates smoothly and efficiently. Inviting team members and assigning them specific roles is essential to distribute responsibilities according to expertise and function.
+
+## Benefits of Team Invitations
+
+- **Specialization**: Assign roles based on expertise, allowing team members to focus on their strengths.
+- **Security**: Limit access to sensitive areas by assigning roles that only allow what members need to see or edit.
+- **Efficiency**: Streamline operations by having clear responsibilities and permissions, which reduces confusion and speeds up processes.
+- **Collaboration**: Foster a collaborative environment where team members can work together on aspects relevant to their roles.
+
+## Team Roles and Permissions
+
+Below is a table outlining the various roles available in Liteflow and their corresponding permissions. This can help you decide which roles to assign to your team members.
+
+| Role             | Permissions                                           | Best For                              |
+| ---------------- | ----------------------------------------------------- | ------------------------------------- |
+| Financial        | Access trade activity and offers on the platform      | Financial oversight and management    |
+| Developer        | Create webhooks and custom plugins                    | Technical development and integration |
+| Management       | Access and edit collections, drops, assets, and users | Overall platform management           |
+| Settings         | Update the settings of the platform                   | Platform configuration                |
+| Collection Owner | Set royalties for the collection they own             | Managing individual collections       |
+
+![Team Member Roles](/mnt/data/Screenshot 2024-02-07 at 14.58.52.png)
+
+## Inviting a Team Member
+
+<Callout type="warning">
+  By carefully selecting roles for each team member, you ensure that everyone
+  has the access they need to perform their duties without compromising the
+  security and integrity of the platform.
+</Callout>
+
+To invite a new team member:
+
+<Steps>
+
+### Go to Team Management
+
+Navigate to the 'Team' section on the Liteflow dashboard.
+
+### Add a Member
+
+Click the "+ Add member" button to start the invitation process.
+
+### Enter Details
+
+In the 'Add an admin' dialog, enter the wallet address and email of the team member you are inviting.
+
+### Assign Roles
+
+Check the appropriate roles based on the permissions you wish to grant to the new member.
+
+### Send Invitation
```

Comment:
We are missing the last part that is: from the user invited what do they receive and how?

From the flow I understand what I'm doing as a platform manager by inviting someone, but currently I have no idea how the invitation looks like, how it will be send, etc

Missing end flow (cf comment from Hodooi on Royalties that is similar)

Reply:
I added a section in the last step https://github.com/liteflow-labs/liteflow-js/pull/223